packages/playground/website/src/components/Toast/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import './css/toast.css';
+import useQuery from '../../hooks/useQuery';
+
+const Toast: React.FC = () => {
+	const onBoarding = useQuery('onBoarding');
+	const [showToast, setShowToast] = React.useState(false);
+	const TOAST_MESSAGE =
+		'All of your modification are private, and will be lost after a page refresh!';
+
+	const showToastMessage = (message: string): void => {
+		/**
+		 * @param message: string, options: object
+		 * @see https://fkhadra.github.io/react-toastify/introduction/
+		 */
+		toast.info(message, {
+			autoClose: 20000,
+			position: toast.POSITION.BOTTOM_CENTER,
```

Comment:
let's add `hideProgressBar` here - it's a bit distracting to see it move around

Reply:
sure, will add here this prop. Thank you!